docs/src/Button.doc.js

```diff
@@ -5,6 +5,8 @@ import PropTable from './components/PropTable.js';
 import Example from './components/Example.js';
 import Combination from './components/Combination.js';
 import PageHeader from './components/PageHeader.js';
+// To be used when converting Button docs
```

Comment:
Totally understand the desire to not lose this work, but rather than leave the commented-out code here, let's revert the changes on this branch and commit them on a separate branch instead. That way the codebase remains clean in the interim, and the changes in this PR are more clear.